src/mudlet.cpp

```diff
@@ -50,6 +50,7 @@
 #include "dlgTriggerEditor.h"
 #include "VarUnit.h"
 
+#include "TChar.h"
```

Comment:
:information_source: Given that this line is also in the class header file ***one*** of these applies:
* you do not need it here
* you could replace the `#include` in the header file with a forward class declaration, i.e. `class TChar;`